source/pacboy/pacboy.sh

```diff
@@ -200,11 +196,11 @@ for argument in "${arguments[@]}"; do
       *:) raw_argument=("${argument%:}") ;;
      *:i) raw_argument=(mingw-w64-i686-${argument%:i}) ;;
      *:x) raw_argument=(mingw-w64-x86_64-${argument%:x}) ;;
-     *:m) raw_argument=(mingw-w64-x86_64-${argument%:m} mingw-w64-i686-${argument%:m}) ;;
+     *:m) raw_argument=(mingw-w64-x86_64-${argument%:m} mingw-w64-i686-${argument%:m}) ;;  # deprecated
```

Comment:
Maybe print a message about deprecation on the use?

Reply:
sure, done